src/main/resources/application.properties

```diff
@@ -0,0 +1,23 @@
+spring.datasource.url=jdbc:h2:mem:music
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=
```

Comment:
Bad practice to store your/database credentials in a repository. Try to use environment variables.